### PR TITLE
Minor changes to Portuguese Brazilian strings

### PR DESCRIPTION
**Pull request by bitcoinuser**

### CODE REVIEW

File: src/common/translations/ptBR [hunk 1]
```diff
@@ -179,12 +179,12 @@
         "selectAddress": "Selecione um endereço",
         "confirmNotice": {
           "one": "Confirme que você deseja criar apelidos para menos de 1 carteira.",
```
Comment (bitcoinuser) [inline src/common/translations/ptBR:181]:
```suggestion
          "one": "Confirme que você deseja criar apelidos para a carteira 1 abaixo.",
```

File: src/common/translations/ptBR [hunk 2]
```diff
@@ -178,13 +178,13 @@
...
-          "one": "Confirme que você deseja criar apelidos para menos de 1 carteira.",
-          "other": "Confirme que você deseja criar apelidos para abaixo de {{count}} carteiras."
+          "one": "Confirme que você deseja criar apelidos para a carteira 1 abaixo.",
+          "other": "Confirme que você deseja criar apelidos para as {{count}} carteiras abaixo."
         }
       },
       "rnsStatus": {
-        "pending": "Registro aguardando confirmação.",
-        "success": "Sucesso! Seu apelido é criado.",
+        "pending": "Aguardando confirmação do registro.",
```
Comment (bitcoinuser) [inline src/common/translations/ptBR:186]:
```suggestion
        "pending": "O registro está sendo confirmado.",
```

File: src/common/translations/ptBR [hunk 3]
```diff
@@ -178,13 +178,13 @@
         "nameAvailable": "O nome digitado está disponível",
         "selectAddress": "Selecione um endereço",
         "confirmNotice": {
-          "one": "Confirme que você deseja criar apelidos para menos de 1 carteira.",
-          "other": "Confirme que você deseja criar apelidos para abaixo de {{count}} carteiras."
+          "one": "Confirme que você deseja criar apelidos para a carteira 1 abaixo.",
```
Comment (solangegueiros) [inline src/common/translations/ptBR:181]:
```suggestion
          "one": "Confirme que você deseja criar apelidos para a carteira abaixo.",
```

**Comment by solangegueiros** [comment]
@chrisli30 and @imstar15 , why the ptBR file do not have the extension .json, like the others translation files?

**Comment by chrisli30** [comment]
> @chrisli30 and @imstar15 , why the ptBR file do not have the extension .json, like the others translation files?

That's a very good point. @bitcoinuser can you make it .json the same as other files?

**Comment by bitcoinuser** [comment]
> > @chrisli30 and @imstar15 , why the ptBR file do not have the extension .json, like the others translation files?
> 
> That's a very good point. @bitcoinuser can you make it .json the same as other files?

Hi people, I'm kind of a newbie. I just wanted to help translating. I don't know how to make it a .json. If someone can help on this I appreciate. Thanks.

**Comment by imstar15** [comment]
@bitcoinuser You can rename this file, and commit again.

**Comment by bitcoinuser** [comment]
> @bitcoinuser You can rename this file, and commit again.

Hi @solangegueiros could you please do this for me? thanks.

File: src/common/translations/ptBR.json [hunk 4]
```diff
@@ -178,13 +178,13 @@
...
-          "one": "Confirme que você deseja criar apelidos para menos de 1 carteira.",
-          "other": "Confirme que você deseja criar apelidos para abaixo de {{count}} carteiras."
+          "one": "Confirme que você deseja criar apelidos para a carteira abaixo.",
+          "other": "Confirme que você deseja criar apelidos para as {{count}} carteiras abaixo."
         }
       },
       "rnsStatus": {
-        "pending": "Registro aguardando confirmação.",
-        "success": "Sucesso! Seu apelido é criado.",
+        "pending": "O registro está sendo confirmado.",
```
Comment (bitcoinuser) [inline src/common/translations/ptBR.json:186]:
```suggestion
        "pending": "O processo de registro está sendo confirmado.",
```

File: src/common/translations/ptBR.json [hunk 5]
```diff
@@ -178,13 +178,13 @@
...
-          "one": "Confirme que você deseja criar apelidos para menos de 1 carteira.",
-          "other": "Confirme que você deseja criar apelidos para abaixo de {{count}} carteiras."
+          "one": "Confirme que você deseja criar apelidos para a carteira abaixo.",
+          "other": "Confirme que você deseja criar apelidos para as {{count}} carteiras abaixo."
         }
       },
       "rnsStatus": {
-        "pending": "Registro aguardando confirmação.",
-        "success": "Sucesso! Seu apelido é criado.",
+        "pending": "O processo de registro está sendo confirmado.",
```
Comment (bitcoinuser) [inline src/common/translations/ptBR.json:186]:
```suggestion
        "pending": "Registro em processo de confirmação.",
```

File: src/common/translations/ptBR.json [hunk 6]
```diff
@@ -178,13 +178,13 @@
...
-          "one": "Confirme que você deseja criar apelidos para menos de 1 carteira.",
-          "other": "Confirme que você deseja criar apelidos para abaixo de {{count}} carteiras."
+          "one": "Confirme que você deseja criar apelidos para a carteira abaixo.",
+          "other": "Confirme que você deseja criar apelidos para as {{count}} carteiras abaixo."
         }
       },
       "rnsStatus": {
-        "pending": "Registro aguardando confirmação.",
-        "success": "Sucesso! Seu apelido é criado.",
+        "pending": "Registro em processo de confirmação.",
```
Comment (bitcoinuser) [inline src/common/translations/ptBR.json:186]:
```suggestion
        "pending": "Registro aguardando confirmação.",
```

**Comment by imstar15** [comment]
Hi, There are some conflicting files.
Please resolve it.

@bitcoinuser

**Comment by bitcoinuser** [comment]
> Hi, There are some conflicting files.
> Please resolve it.
> 
> @bitcoinuser

@solangegueiros Não sei como fazer para resolver esse conflito. Pode me ajudar?

**Comment by chrisli30** [comment]
> > Hi, There are some conflicting files.
> > Please resolve it.
> > @bitcoinuser
> 
> @solangegueiros Não sei como fazer para resolver esse conflito. Pode me ajudar?

Hi @bitcoinuser @solangegueiros , could you resolve the conflicts? We are ready to merge this file.

**Pull request closed without merging**